docs/release/flux.md

```diff
@@ -0,0 +1,173 @@
+# Flux release spec
+
+The Flux project repository [fluxcd/flux2](https://github.com/fluxcd/flux2) contains
+the Flux command-line tool source code and the Kubernetes manifests for
+bundling the [Flux controllers](controllers.md) into a distributable package.
+
+## Release versioning
+
+Flux is released by following the [semver](https://semver.org/) conventions:
+
+- `vX.Y.Z-RC.W` release candidates e.g. `v2.0.0-RC.1`
+- `vX.Y.Z` stable releases e.g. `v2.0.0`
+
+The Flux project maintains release branches for the most recent three minor releases
+e.g. `release-2.0`, `release-2.1` and `release-2.2`.
```

Comment:
```suggestion
- `vX.Y.Z-RC.W` release candidates e.g. `v2.0.0-rc.1`
- `vX.Y.Z` stable releases e.g. `v2.0.0`

The Flux project maintains release branches for the most recent three minor releases
e.g. `release/2.0.x`, `release/2.1.x` and `release/2.2.x`.
```

- We have tagged our RC with `-rc.X`, so lets use that as a proper example.
- I prefer `/` "namespacing" on repositories where features are also developed in branches, and `-` is used in those names. This also eases things like branch protection and automation without creating false positives.